/**:
  ros__parameters:
    speed_to_erpm_gain: 1950.0
    speed_to_erpm_offset: 0.0
    steering_angle_to_servo_gain: -1.0
    steering_angle_to_servo_offset: 0.5
    battery_cells: 4
    accel_to_current_gain: 1.0